lib/generators/rom/commands/templates/create.rb.erb

```diff
@@ -0,0 +1,8 @@
+class <%= class_name.singularize.camelcase %>Commands::Create < ROM::Commands::Create<%= "[:#{options[:adapter]}]" if options[:adapter] %>
```

Comment:
This can't be inlined unless the module is already defined and I believe it's not? So:

``` erb
module <%= class_name.singularize.camelcase %>Commands
  class Create < ROM::Commands::Create<= "[:#{options[:adapter]}]" %>

# ...
end
```